### ISSUE TITLE
When i am training the ArbRCAN model, the training loss is so large. Is it a normal situation?

### ISSUE THREAD

**Issue by RayTan183** [LongguangWang/ArbSR#24]
When i am training the ArbRCAN model, the training is so large that it is a normal situation. Sometime the loss so so large that skip the batch. And i tried to test by using the middle weights such as 70.pth. But the result even worse than the original pre-trained model.

**Comment by RayTan183** [comment]
![image](https://github.com/The-Learning-And-Vision-Atelier-LAVA/ArbSR/assets/145185430/62752815-e5bd-44c1-8847-3081f9f59f11)

**Comment by hjdihs** [comment]
Is it a normal situation?
